src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/MaterialFontsOverride.xaml

```diff
@@ -7,5 +7,7 @@
 					xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
 					mc:Ignorable="android ios wasm">
 
-	<FontFamily x:Key="MaterialFontFamily">ms-appx:///Assets/Fonts/Material/RobotoMono-VariableFont_wght.ttf#Roboto Mono</FontFamily>
+	<FontFamily x:Key="MaterialLightFontFamily">ms-appx:///Assets/Fonts/Material/Roboto-Light.ttf#Roboto</FontFamily>
+	<FontFamily x:Key="MaterialMediumFontFamily">ms-appx:///Assets/Fonts/Material/Roboto-Medium.ttf#Roboto</FontFamily>
+	<FontFamily x:Key="MaterialRegularFontFamily">ms-appx:///Assets/Fonts/Material/Roboto-Regular.ttf#Roboto</FontFamily>
```

Comment:
This works? Don't we base FontFamily on the part of the string that comes after the #? 

If we set a control's FontFamily to just "Roboto", how would it know which of these tree to use since they all end with `#Roboto`

Reply:
Depends on the FontFamily file (otf, ttf,...) and the way is was created, for UWP having anything else other than Roboto does not load the correct FontFamily. I did multiple test (screenshot comparison, superposition,...) and that's the combination that is working for these Roboto font family files and UWP.

For example, having <FontFamily x:Key="MaterialLightFontFamily">ms-appx:///Assets/Fonts/Material/Roboto-Light.ttf#**Roboto Light**</FontFamily> is not working for UWP in this case.

Reply:
strange, well as long as it works on all paltforms that's good

Reply:
I need to continue my testing for some platforms before merging this PR as I had issues with my android emulators and WASM IISEspress before pushing my latest changes

Reply:
@ajpinedam If you have some time and would like to test macOS it will help me (thanks in advance!)

Reply:
Latest status for the tests:
- [X] UWP
- [X] WASM
- [X] Android
- [X] iOS ( with <string>Fonts/Roboto-Light.ttf</string>instead of <string>Fonts/Material/Roboto-Light.ttf</string>)
? MacOS (will double check with @ajpinedam )

**Interesting find for iOS**, seems like the Font files shouldn't be in a subfolder... as for some reason it's not loading correctly the font:
![Material FontFamily iOS issue with subfolder](https://user-images.githubusercontent.com/16295702/143088313-368a03e8-65c8-4d53-a51d-2c350031592e.png)